frontend/src/screens/Start.tsx

```diff
@@ -54,7 +75,7 @@ export default function Start() {
       setButtonText("Login");
       setUnlockPassword("");
       return;
-    }, 180000); // wait for 3 minutes
+    }, 30000); // wait for 30 seconds
```

Comment:
Note: this is different than the setup page, because the user has more data and syncing takes longer. But it should still be ok